app/assets/stylesheets/application.scss

```diff
@@ -74,33 +68,36 @@
 }
 
 body {
-  background: #222222;
-  font-family: 'Nexa Light', sans-serif;
-  color: white;
+  background: $background-color;
+  font-family: $nexa-font;
+  color: $font-color;
 }
 
 #title {
-  background: #222222;
+  background: $background-color;
   text-align: center;
 }
+
 h1, h2, h3, h4, h5, h6 {
-  font-family: 'Nexa Light', sans-serif;
-  color: white;
+  font-family: $nexa-font;
```

Comment:
Properties should be ordered color, font-family, text-align<br>Each selector in a comma sequence should be on its own line